src/modules/formBuilder/components/itemEditor/FormEditorItemProperties.tsx

```diff
@@ -142,6 +142,12 @@ const FormEditorItemProperties: React.FC<FormEditorItemPropertiesProps> = ({
     }
   }, [hiddenValue, setValue]);
 
+  useEffect(() => {
```

Comment:
And, well, to contradict myself, here I switched to using an effect instead of the previous callback (see `ManageEnableWhen` below) which wasn't working, though I haven't deeply investigated why not, and would be happy to put some more time into it if we think it's worthwhile.

Possibly it might relate to us mix and matching  rhf's `setValue` with `onChange`, which the docs explain we should avoid. https://www.react-hook-form.com/api/usecontroller/controller/